# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/js.msg
# Commit: 920da79f43fc
# Full Hash: 920da79f43fcf5802ebd5eb2bf610a18e821dacc
# Author: Andy Wingo <wingo@igalia.com>
# Date: 2020-03-19 03:53:34
# Regressor Bug: 1620197
# File Overlap Count: 1
# Description:
#   Bug 1620197 - Enable multiple results from WebAssembly functions r=lth
#   
#   This patch enables multi-value calls and returns, adding some tests, and
#   conditionally disabling a couple expect-fail reftests that now pass.
#   
# ==============================================================================

diff -r 98d7f9e4839f -r 920da79f43fc js/src/js.msg
--- a/js/src/js.msg	Wed Mar 18 17:12:14 2020 +0000
+++ b/js/src/js.msg	Wed Mar 18 16:36:45 2020 +0000
@@ -429,6 +429,8 @@
 MSG_DEF(JSMSG_WASM_MISSING_MAXIMUM,    0, JSEXN_TYPEERR,     "'shared' is true but maximum is not specified")
 MSG_DEF(JSMSG_WASM_GLOBAL_IMMUTABLE,   0, JSEXN_TYPEERR,     "can't set value of immutable global")
 MSG_DEF(JSMSG_WASM_NULL_REQUIRED,      0, JSEXN_TYPEERR,     "nullref requires a null value")
+MSG_DEF(JSMSG_WASM_MULTIPLE_RESULT_EXIT_UNIMPLEMENTED, 0, JSEXN_TYPEERR, "calling JavaScript functions with multiple results from WebAssembly not yet implemented")
+MSG_DEF(JSMSG_WASM_MULTIPLE_RESULT_ENTRY_UNIMPLEMENTED, 0, JSEXN_TYPEERR, "calling WebAssembly functions with multiple results from JavaScript not yet implemented")
 
 // Proxy
 MSG_DEF(JSMSG_BAD_TRAP_RETURN_VALUE,   2, JSEXN_TYPEERR,"trap {1} for {0} returned a primitive value")